src/goto-programs/show_goto_functions.h

```diff
@@ -15,6 +15,12 @@ class goto_functionst;
 class namespacet;
 class goto_modelt;
 
+#define GOTO_SHOW_FUNCTIONS_OPTIONS \
+  "(show-goto-functions)"
+
+#define GOTO_SHOW_FUNCTIONS_HELP \
+  " --show-goto-functions         show goto program\n"
+
```

Comment:
I believe I have messed up when introducing the same for `GOTO_CHECK_*` - the macros sound Yoda-like. How about a general principle (and thus to be applied to GOTO_CHECK...): `OPT_...` for the options string, and `HELP_...` for the help output?